README.md

```diff
@@ -342,6 +342,61 @@ simply add `--debug` flag to any command.
 
 [[back to top](#databricks-labs-ucx)]
 
+## Table Migration Workflow
```

Comment:
Can you install markdown plugin for intellij and regenerate table of contents?

Please also update other references of table migration to point to this section instead - they're pointing to table migration commands at the moment.